packages/flutter/lib/src/services/text_input.dart

```diff
@@ -571,6 +563,15 @@ class TextInputConfiguration {
   /// {@endtemplate}
   final bool enableSuggestions;
 
+  /// Whether to a user can change its selection.
+  ///
+  /// This flag only affects iOS VoiceOver. On Android Talkback, the selection
+  /// change is sent through semantics actions and are directly disabled from
```

Comment:
What is "directly disabled" here, the selection change?  If so then "are" should be "is".